### PR TITLE
252 docs system requirements

### PR DESCRIPTION
**Pull request by cherokee94**
@arufanov 

1. Can you provide more specific system requirements for AXION? (I don't think that nexus's are appropriate)
2. What DB we specify for use?
3. I don't think that "Dedicated operating system user account" section is required...

### CODE REVIEW

File: docs/modules/installation/pages/system_requirements.adoc [hunk 1]
```diff
@@ -1,11 +1,65 @@
...
-* host operation system
-* "Dedicated Operating System User Account
-"
+Make sure that your system meets our recommended hardware and software requirements.
 
+== Hardware requirements
 
-Could you review https://help.sonatype.com/repomanager3/product-information/system-requirements we are interested at sections "Host Operating System" & "Dedicated Operating System User Account
-" & " Java". Also could you add "hardware requirements" based on cpu, memory and storage section. Could you put it first.
+Description of recommended hardware for *AXION*.
```
Comment (arufanov) [inline docs/modules/installation/pages/system_requirements.adoc:7]: @cherokee94 
* Axion (https://en.wikipedia.org/wiki/Axion) is not acronym. How should we write is correctly? *axion* or *Axion*?

**Review by arufanov — COMMENTED** [review]
@yurichervoniy 
* lest discuss one conversation about *AXION* and I will accept it.

File: docs/modules/installation/pages/system_requirements.adoc (hunk 1 above)
Comment (cherokee94) [reply docs/modules/installation/pages/system_requirements.adoc:7]: @arufanov we can keep it "classic" for product name in documentation – Axion

**Review by arufanov — APPROVED** [review]

**Comment by arufanov** [comment]
@cherokee94 
* has been approved, you can merge

**Comment by cherokee94** [comment]
@arufanov need more rights to merge )

**Pull request merged**